packages/bvaughn-architecture-demo/components/console/MessageHoverButton.module.css

```diff
@@ -1,11 +1,13 @@
 .AddCommentButton,
 .ConsoleMessageHoverButton {
+  font-family: "Inter", sans-serif;
+  font-weight: 600;
   padding: 0 0.125rem;
   position: absolute;
   left: 0.25rem;
   top: 0.25rem;
   height: 1.5rem;
-  border-radius: 0.25rem;
+  border-radius: 3rem;
```

Comment:
I'm not a fan of this change. I think the button looked significantly better before.

### Before
![image](https://user-images.githubusercontent.com/29597/183933071-1bed3cec-f03a-47ca-aaad-dca243f566f5.png)

### After
![image](https://user-images.githubusercontent.com/29597/183933097-f5065a9b-0a57-4dd9-bc89-b1cdef70286a.png)

The new layout feels cramped and vertical alignment looks a bit off. (To be fair it looks a bit misaligned in both of them, but it feels less noticeable in the less cramped before style.)

Reply:
Just pushed up a revision of this that's a middle ground and matches what we currently do. I'd like the design to be as similar as possible to our current aesthetic for now and we can follow-up and make tweaks afterwards.